index.js

```diff
@@ -15,6 +15,16 @@ let reverse = (s) => {
   return o;
 };
 
+exports.createS3UploadParams = (bucketname, data) => {
+  // The synctoken is unix epoch time, using a regex is faster than parsing JSON :)
+  let regexToken = data.match(/syncToken[^0-9]+([0-9]+)/)[1];
+  return {
+    "Bucket": bucketname,
```

Comment:
Codacy has a fix for the issue: [Replace `"Bucket"` with `Bucket`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
    Bucket: bucketname,
```